### PR TITLE
file field

### PR DESCRIPTION
**Pull request by jjidt**
sets up file field, still needs styling

### CODE REVIEW

File: web/src/client/js/components/DocumentFields/DocumentFieldComponent.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ const DocumentFieldComponent = ({
     }
   }, [isNewField])
 
-  const dataField = field.type !== FIELD_TYPES.TEXT && field.type !== FIELD_TYPES.IMAGE
+  const dataField = field.type !== FIELD_TYPES.TEXT && field.type !== FIELD_TYPES.IMAGE && field.type !== FIELD_TYPES.FILE
```
Comment (dheniges) [inline web/src/client/js/components/DocumentFields/DocumentFieldComponent.js]: this should be a whitelist

**Review by dheniges — CHANGES_REQUESTED** [review]
Just one change in the client that I stumbled across the other day, then this looks good to go. I'm ignoring all the copypasta for image field -> file field on the client :)

File: web/src/client/js/components/DocumentFields/DocumentFieldComponent.js (hunk 1 above)
Comment (jcontonio) [reply web/src/client/js/components/DocumentFields/DocumentFieldComponent.js]: All set

**Review by dheniges — APPROVED** [review]
:+1:

**Pull request merged**